src/mds/MDCache.h

```diff
@@ -404,7 +404,7 @@ class MDCache {
   void project_rstat_frag_to_inode(nest_info_t& rstat, nest_info_t& accounted_rstat,
 				   snapid_t ofirst, snapid_t last, 
 				   CInode *pin, bool cow_head);
-  void broadcast_quota_to_client(CInode *in);
+  void broadcast_quota_to_client(CInode *in, entity_inst_t *exclude_inst = NULL);
```

Comment:
use client_t instead of entity_inst_t

Reply:
Ok, client_t will also do the work. I will update it. But why client_t?

Reply:
client_t comparison is  cheaper